data-jdbc/src/test/groovy/io/micronaut/data/jdbc/oraclexe/OracleXERepositorySpec.groovy

```diff
@@ -207,4 +209,54 @@ class OracleXERepositorySpec extends AbstractRepositorySpec implements OracleTes
             cleanupBooks()
     }
 
+    @PendingFeature
+    void "test update returning book"() {
+        given:
+            setupBooks()
+        when:
+            def book = bookRepository.findByTitle("Pet Cemetery")
+            book.title = "Xyz"
+            Book newBook = bookRepository.updateReturning(book)
+            book.title = "old"
```

Comment:
This line can be removed I think, just to clear up potential confusion (at least for me).

Reply:
That is testing that the method doesn’t return the previous entity instance. That should fail for ordinary update methos

Reply:
After assigning `book.title = "old"` nothing is being done with `book` variable so I don't understand why it was needed. But it's not important.
Why is this test `@PendingFeature`? I tried running it and got the error 

```
Error executing SQL Query: ORA-00925: missing INTO keyword

https://docs.oracle.com/error-help/db/ora-00925/
io.micronaut.data.exceptions.DataAccessException: Error executing SQL Query: ORA-00925: missing INTO keyword
```

Reply:
Oracle needs more work

Reply:
If the update returns the same instance modifying the book variable will break the test